ckanext/ontario_theme/templates/internal/group/index.html

```diff
@@ -19,16 +19,16 @@ <h1 class="ontario-h1">{% block page_heading %}{{ _('Groups') }}{% endblock %}</
 
   {% block groups_search_form %}
     <!-- arguments for DS search bar -->
-    {% set sr_input_text = _("Search") %}
+    {% set input_text = _("Search groups") %}
     {% set sr_submit_text = _("Search") %}
-    {% set placeholder = _('Search groups') %}
     {% set query = c.q %}
     <!-- arguments for search form -->
     {% set form_id = 'group-search-form' %}
     {% set type = 'group' %}
-    {% set searchbar_title = _('Search groups') + _(':') %}
-    {% set hidden_searchbar_title = _('Search groups') %}
-    {% snippet 'snippets/search_form.html', searchbar_title=searchbar_title, hidden_searchbar_title=hidden_searchbar_title, form_id=form_id, type=type, query=query, sorting_selected=c.sort_by_selected, count=c.page.item_count, placeholder=placeholder, sr_input_text=sr_input_text, sr_submit_text=sr_submit_text, show_empty=request.params, no_bottom_border=true if c.page.items, sorting = [(_('Name Ascending'), 'name asc'), (_('Name Descending'), 'name desc')] %}
+    {% set searchbar_title = input_text + _(':') %}
+    <form {% if form_id %}id="{{ form_id }}" {% endif %}class="search-form{% if no_bottom_border %} no-bottom-border{% endif %}" method="get" data-module="select-switch" novalidate>
```

Comment:
why is `if form_id` condition here? Looks like this condition will always be true since `form_id` is set on line 26.